pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -1720,7 +1720,7 @@ default-package-overrides:
   - postgresql-libpq ==0.9.4.2
   - postgresql-orm ==0.5.1
   - postgresql-simple ==0.6.2
-  - postgrest ==7.0.0
```

Comment:
These are versions from the latest stackage LTS, so we don't change them by hand.   I imagine the next version of stackage will bump `postgrest` to 7.0.1, so we can just wait for that.

Reply:
Oh, I didn't know that, thank you. but the rest should be updated like I did ?

Reply:
Ah, no, I think everything else in this PR should happen automatically as well, so I think you can go ahead and close this.

Check out https://github.com/NixOS/nixpkgs/pull/92461 over the next couple days.

Reply:
oh, alright. thanks for the info and sorry for wasting your time. among my first PR.